templates/layout/page--node--preview.html.twig

```diff
@@ -0,0 +1,114 @@
+{#
```

Comment:
What's the reasoning for this template? I don't see any difference between this and our page.html.twig template (but I'm probably just blind).

Reply:
@markconroy 
In the preview we were getting the breadcrumb and title blocks printed,  so this just strips them out. 
ie the blocks  with the  `{% if has_breadcrumb and not node %}` argument. 
I think the more elegant solution would be to make the page.html.twig recognise the preview , but I couldnt figure that out I'm afraid. Wanna have a go?

Reply:
Ok. Well let's stick this template in for now, since it meets that use case, and we can see about adding a variable somehow to get this done in the main page.html.twig template later.